_component_design/time-picker.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Time Picker
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: Enable users to select a specific time or window from a sequential list. The Time Picker is often accompanied by a date picker for the purpose of creating a date range filter.
```

Comment:
I think I wrote this so i'm editing myself, I would prepend "Enable users" with the thing that enables.

Suggested Edit: "Time pickers enable users to select a specific time or window from a sequential list. The time picker component is often accompanied by a date picker for the purpose of creating a date range filter."

Also, perhaps we hyper-link the "...date picker..." [date picker]({{site.baseurl}}/components/date-picker.html)

Reply:
excellent feedback.  will do.

Reply:
looks like the front-matter yaml can't contain links, so the cross-ref might not work out.

Reply:
gotcha. No worries.